### DISCUSSION TITLE
Problems with Primeng components

### DISCUSSION THREAD

**Discussion by juansanchez1990** [primefaces/community#2947] [PrimeNG]
Why is it that all the Prime NG components in my Angular project are designed differently than those in the documentation?

I've already placed the styles in angular.json, I've already put them in the style files and it still doesn't show them as they appear on the official documentation page.

**Comment by qwadrox** [comment]
@juansanchez1990 
What do you mean by not appearing as in the docs? The size is different? 
If yes, read the docs: https://primeng.org/theming#scale
On the docs site, the global `font-size` is set to 14px. 
If not, can you share a screenshot or something more descriptive?

**Reply by juansanchez1990** [reply]
The components that appear on the official page do not appear the same when I implement them in an Angular project.

The style is not minimalist as they appear on the official page.

**Reply by qwadrox** [reply]
@juansanchez1990 Well without further information, screenshot and example app the community and the mainteners won't be able to help you.

**Reply by juansanchez1990** [reply]
![selectores](https://github.com/user-attachments/assets/4658f3d6-d0a7-46b1-b093-01e953c23fac)

On the left is the selector for my Angular project, on the right is the multiselector from the official Prime NG page

If you look closely, the multiselector for my project is very different from the one shown in the example on the official Prime NG page

**Reply by qwadrox** [reply]
Yes, I can see. But this does not seem to be related to PrimeNG. I advise you to check what styles affect that extra padding on the left with the browser developer tools. Also, make sure that if you use any CSS library or other Component Library, their styles do not interfere with each other. For instance with bootstrap you need this extra step: https://primeng.org/guides/csslayer#bootstrap

**Reply by juansanchez1990** [reply]
I already tried it but it doesn't work, everything always comes out different.

**Reply by SippieCup** [reply]
The reason why is that primeNG on the documentation page still have a whole lot of CSS code built into the docs to get the styling of their docs page.  these additional CSS classes and such interact deeply with the v18 content.

https://github.com/primefaces/primeng/tree/v18-prod/src/assets/showcase/styles/layout

That said, there are still a few other things that are just incorrect and faked on the docs or theres some wierd hackery going on. Cards, for example, don't display titles. panel menus have multiple labels, etc.
